integration-tests/billing/tests/supplementary/SB2.js

```diff
@@ -124,43 +120,24 @@ experiment('supplementary ref: SB2', () => {
           licence = invoice.billingInvoiceLicences[0];
         });
 
-        test('has the correct licence name', async () => {
-          expect(licence.licenceHolderName.lastName).to.equal('Testerson');
-          expect(licence.licenceHolderName.firstName).to.equal('John');
-          expect(licence.licenceHolderName.title).to.equal('Mr');
-        });
-
-        test('has the correct licence holder address', async () => {
-          expect(omit(licence.licenceHolderAddress, 'id')).to.equal({
-            town: 'Testington',
-            county: 'Testingshire',
-            country: 'UK',
-            postcode: 'TT1 1TT',
-            addressLine1: 'Big Farm',
-            addressLine2: 'Windy road',
-            addressLine3: 'Buttercup meadow',
-            addressLine4: null
-          });
-        });
-
-        test('has 3 transactions', async () => {
-          expect(licence.billingTransactions.length).to.equal(3);
+        test('has 6 transactions', async () => {
+          expect(licence.billingTransactions.length).to.equal(6);
         });
 
         test('there are 2 debit', async () => {
-          const transactions = licence.billingTransactions.filter(tx => tx.isCredit === false);
+          const transactions = licence.billingTransactions.filter(tx => tx.isCredit === false && tx.chargeType === 'standard');
           expect(transactions).to.have.length(2);
         });
 
         test('there is a credit', async () => {
-          const transaction = licence.billingTransactions.find(tx => tx.isCredit === true);
+          const transaction = licence.billingTransactions.find(tx => tx.isCredit === true && tx.chargeType === 'standard');
           expect(transaction).to.exist();
         });
 
         experiment('the first debit transaction', () => {
           let transaction;
           beforeEach(async () => {
-            transaction = licence.billingTransactions.find(tx => tx.isCredit === false && tx.description === 'CE3');
+            transaction = licence.billingTransactions.find(tx => tx.transactionKey === '1e1fee0c4146c314e12f5f9067b25b24');
```

Comment:
Doesn't this make the 'has a stable transaction key' test redundant?

Reply:
Fixed